coderd/wsbuilder/wsbuilder.go

```diff
@@ -363,23 +363,20 @@ func (b *Builder) buildTx(authFunc func(action policy.Action, object rbac.Object
 	var workspaceBuild database.WorkspaceBuild
 	err = b.store.InTx(func(store database.Store) error {
 		err = store.InsertWorkspaceBuild(b.ctx, database.InsertWorkspaceBuildParams{
-			ID:                workspaceBuildID,
-			CreatedAt:         now,
-			UpdatedAt:         now,
-			WorkspaceID:       b.workspace.ID,
-			TemplateVersionID: templateVersionID,
-			BuildNumber:       buildNum,
-			ProvisionerState:  state,
-			InitiatorID:       b.initiator,
-			Transition:        b.trans,
-			JobID:             provisionerJob.ID,
-			Reason:            b.reason,
-			Deadline:          time.Time{}, // set by provisioner upon completion
-			MaxDeadline:       time.Time{}, // set by provisioner upon completion
-			TemplateVersionPresetID: uuid.NullUUID{
-				UUID:  uuid.Nil,
-				Valid: false,
-			},
+			ID:                      workspaceBuildID,
+			CreatedAt:               now,
+			UpdatedAt:               now,
+			WorkspaceID:             b.workspace.ID,
+			TemplateVersionID:       templateVersionID,
+			BuildNumber:             buildNum,
+			ProvisionerState:        state,
+			InitiatorID:             b.initiator,
+			Transition:              b.trans,
+			JobID:                   provisionerJob.ID,
+			Reason:                  b.reason,
+			Deadline:                time.Time{},     // set by provisioner upon completion
+			MaxDeadline:             time.Time{},     // set by provisioner upon completion
+			TemplateVersionPresetID: uuid.NullUUID{}, // TODO (sasswart): add this in from the caller
```

Comment:
TODO

Reply:
mind if we defer this one? we need it for prebuilds so it won't get lost.